scripts/scripts.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module '@gimenete/type-writer';
+declare module 'table-builder';
```

Comment:
[❓ ] I'm not sure if this is the right approach when depending on 3rd party dependencies which are not exposing its own declaration. Is this correct?

Reply:
Not exactly, this would make everything have type of `any` or throw errors because there are no exported properties.

You'll need to document all exports from the imported modules used in the codebase

Reply:
Naming for declaration file?
* `scripts/sccript.d.ts`
* `scripts/index.d.ts`
* same but in root level of the project?

Reply:
You could name it `external-deps.d.ts`

Reply:
Not an issue anymore after this agreement:
https://github.com/octokit/webhooks.js/pull/113#discussion_r416975529